modules/ROOT/pages/publishing-a-rest-api.adoc

```diff
@@ -0,0 +1,16 @@
+= Publishing a REST API
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Mule provides a dedicated toolkit to facilitate the development and exposure of REST APIs. APIkit Rest is a simple framework that helps you with API-first development, enforcing good API implementation practices. You can add this toolkit to your instance of Anypoint Studio. +
+
+Rather than spending weeks or months building an API, you can use this toolkit to develop, document, and test your API within a few days or even hours.
+
+Use APIkit to create an API based on RAML (RESTful API Modeling Language) or OAS (OpenAPI Specification) API definition. The toolkit produces a Mule application with an HTTP endpoint and exception strategies and also offers an API console as a design feature only, to test your API while developing.
+
+Learn more about xref:apikit::index.adoc[APIkit].
```

Comment:
"To learn more about APIkit, see <xref>."
But this might not be necessary if we omit or combine topics.
Also, when you have a See Also section, you can move all xrefs to that section rather than putting them in the narrative.